changelog/db.changelog-1.3.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="DEFRA" id="1596009647200-1">
+      <tagDatabase tag="v1.1.0" />
```

Comment:
We cant change this now as its already been deployed to SND but this version should match the file name so `1.3`. The next changelog should be tagged `1.4.0` to match the filename `db.changelog-1.4.xml`.

This is incase we ever wanted to roll back to a previous database version it will make more sense if the versions and the filenames are in sync.

Reply:
Ah my mistake, as the previous ones had `1.0.0` as the version number I bumped the version for the `db.changelog-1.3.xml` to `1.1.0`, but will note that it should match the filename as you said.